src/applications/resources-and-support/hooks/useGetSearchResults.js

```diff
@@ -65,18 +65,45 @@ export default function useGetSearchResults(articles, query, page) {
         ),
 
         // Number of times a keyword is found in the article's introText.
-        keywordsCountsDescription: keywords?.reduce(
+        keywordsCountsIntroText: keywords?.reduce(
           (keywordInstances, keyword) =>
             keywordInstances +
             article.introText.toLowerCase()?.split(keyword)?.length -
             1,
           0,
         ),
 
-        wholePhraseMatchCounts:
+        // Number of times a keyword is found in the article's searchableContent.
+        keywordsCountsContent: keywords?.reduce(
+          (keywordInstances, keyword) =>
+            keywordInstances +
+            article.searchableContent.toLowerCase()?.split(keyword)?.length -
+            1,
+          0,
+        ),
+
+        // Number of times the full query is found in the article's title.
+        wholePhraseMatchCountsTitle:
+          article.title.toLowerCase()?.split(query.toLowerCase())?.length - 1,
```

Comment:
Potential for refactoring since this pattern is repeated a few times

Reply:
Updated